src/main/java/com/suchocki/parkingmeter/entity/Driver.java

```diff
@@ -0,0 +1,51 @@
+package com.suchocki.parkingmeter.entity;
+
+public class Driver {
+	private String licensePlate; // primary key in database
+	private DriverType driverType;
+
+	public Driver() {
+		;
+	}
+
+	public Driver(String licensePlate, DriverType driverType) {
+		this.licensePlate = licensePlate;
+		this.driverType = driverType;
+	}
+
+	// There could be some additional fields like name, surname etc.
```

Comment:
Not there, but some lines above, with all field declaration

Reply:
Niewątpliwie. Miała być to tylko bardziej informacja, uzasadniająca użyteczność tej klasy przy ewentualnych zmianach w przyszłości (np. gdy chcielibyśmy przechowywać więcej informacji na temat kierowców).